misc/python/materialize/checks/scenarios_upgrade.py

```diff
@@ -118,8 +118,15 @@ def base_version(self) -> MzVersion:
         return minor_versions[-4]
 
     def actions(self) -> list[Action]:
+        minor_minus_2 = minor_versions[-2]
+        minor_minus_1 = minor_versions[-1]
+        # TODO(def-) This version was bad, see #23160, can be removed after the next release
```

Comment:
What about adding some `BROKEN_VERSION` like `INVALID_VERSIONS` in `misc/python/materialize/version_list.py`?

Reply:
Or adding it to `INVALID_VERSIONS`